### PR TITLE
Consolidate test_did_rpc.py into test_wallet_rpc.py, make the tests use the wallet RPC client and fix name annotation in create_new_did_wallet()

### PR DESCRIPTION
**Pull request by AmineKhaldi**

### CODE REVIEW

**Review by ytx1991 — APPROVED** [review]

File: .github/workflows/build-test-macos-wallet-did_wallet.yml [hunk 1]
```diff
@@ -95,7 +95,7 @@ jobs:
     - name: Test wallet-did_wallet code with pytest
       run: |
         . ./activate
-        venv/bin/coverage run --rcfile=.coveragerc --module pytest --durations=10  -n 0 -m "not benchmark" tests/wallet/did_wallet/test_did.py tests/wallet/did_wallet/test_did_rpc.py
+        venv/bin/coverage run --rcfile=.coveragerc --module pytest --durations=10  -n 0 -m "not benchmark" tests/wallet/did_wallet/test_did.py
```
Comment (Netting-Starrs) [inline .github/workflows/build-test-macos-wallet-did_wallet.yml:98]: Approved

File: tests/wallet/rpc/test_wallet_rpc.py [hunk 2]
```diff
@@ -759,6 +759,7 @@ def only_ids(trades):
 async def test_did_endpoints(wallet_rpc_environment: WalletRpcTestEnvironment):
     env: WalletRpcTestEnvironment = wallet_rpc_environment
 
+    wallet_1: Wallet = env.wallet_1.wallet
```
Comment (Netting-Starrs) [inline tests/wallet/rpc/test_wallet_rpc.py:762]: wallet_1 added

File: tests/wallet/rpc/test_wallet_rpc.py [hunk 3]
```diff
@@ -768,10 +769,26 @@ async def test_did_endpoints(wallet_rpc_environment: WalletRpcTestEnvironment):
     await generate_funds(env.full_node.api, env.wallet_1, 5)
 
     # Create a DID wallet
-    res = await wallet_1_rpc.create_new_did_wallet(1)
+    res = await wallet_1_rpc.create_new_did_wallet(amount=1, name=None)
```
Comment (Netting-Starrs) [inline tests/wallet/rpc/test_wallet_rpc.py:772]: wallet_1_rpc.create_new_did_wallet(amount=1, name=None)

File: tests/wallet/rpc/test_wallet_rpc.py [hunk 4]
```diff
@@ -768,10 +769,26 @@ async def test_did_endpoints(wallet_rpc_environment: WalletRpcTestEnvironment):
...
+    # Set wallet name
+    new_wallet_name = "test name"
+    res = await wallet_1_rpc.did_set_wallet_name(did_wallet_id_0, new_wallet_name)
+    assert res["success"]
+    res = await wallet_1_rpc.did_get_wallet_name(did_wallet_id_0)
+    assert res["success"]
+    assert res["name"] == new_wallet_name
+    with pytest.raises(ValueError, match="Wallet id 1 is not a DID wallet"):
+        await wallet_1_rpc.did_set_wallet_name(wallet_1.id(), new_wallet_name)
+
```
Comment (Netting-Starrs) [inline tests/wallet/rpc/test_wallet_rpc.py:791]: looks great!

File: tests/wallet/rpc/test_wallet_rpc.py (hunk 3 above)
Comment (Netting-Starrs) [inline tests/wallet/rpc/test_wallet_rpc.py:772]: Great job!

File: tests/wallet/did_wallet/test_did_rpc.py [hunk 5]
```diff
@@ -1,57 +0,0 @@
-import asyncio
```
Comment (Netting-Starrs) [inline tests/wallet/did_wallet/test_did_rpc.py:1]: import

**Pull request merged**